### ISSUE TITLE
Multiple Crawlers Committing to Same Solr Core

### ISSUE THREAD

**Issue by dhildreth** [Norconex/collector-http#414]
I'm wondering if I have a basic misunderstanding here.  I've configured several different crawlers (`<crawler>`) for my different domains (ie. www.myredacteddomain.com, wiki.myredacteddomain.com, support.myredactedomain.com).  I've done this because each domain has a different layout and I'd like to use `StripBeforeTransformer` and `StripAfterTransformer` to narrow in on the content that matters (exclude header, breadcrumb, and footer).  I'd like all of the results of all domains to be committed to a single Solr core.

Now, I don't see any strange issues that stand out to me in the crawler logs.  In fact, I see messages from `[SolrCommitter]` that state documents are being sent into Solr for each of the crawlers.  However, when I goto Solr admin page and attempt to query, I'm only able to see results from the crawler that finished last.

Am I missing something?  Does each crawler need it's own Solr core?  If this is true, I'm going to need to figure out how to apply the different transformers to each site.  I suppose I could use `RestrictTo` for that...

**Comment by essiembre** [comment]
If you try to is to specify a different ``<queueDir>`` for each committer you have for each crawler does it make a difference? Can you share your config to reproduce?

**Comment by dhildreth** [comment]
I got to thinking about this and decided there was probably a better way to go about it.  Instead of separate crawlers, I used `restrictTo` for each of the transformers to the pertinent sites.  Now, things are scanning and behaving as I'd like them to.

For what it's worth, I did not make the `<workDir>` or `<queueDir>` unique to each crawler.  That might've been the key.  I've since dumped that config file.
